src/azure-cli/azure/cli/__main__.py

```diff
@@ -43,9 +45,21 @@ def cli_main(cli, args):
     else:
         telemetry.set_success()
 
+    elapsed_time = timeit.default_timer() - start_time
```

Comment:
Why do we log the time for each command?  I worry this may attract more perf related issues being raised.

Reply:
I think this was brought up in the meeting with @yugangw-msft and  @achandmsft. I think the rational was to help customers notice / point out situations were commands are much slower than usual. I suppose we just saw it as general command related information. 

If perf really is a problem (esp. on the CLI side) moving forward then I think it doesn't hurt to give customers and ourselves more clarity regarding it. If it **really** is a problem then we eventually have to address it.

I guess we could discuss this more when I am back from the conference.